src/fsharp/fsc.fs

```diff
@@ -1706,38 +1701,40 @@ module StaticLinker =
 
 type SigningInfo = SigningInfo of (* delaysign:*) bool * (*signer:*)  string option * (*container:*) string option
 
+let GetSigner(signingInfo) = 
```

Comment:
FCS uses this logic from multiple places so it is factored out into a single function in preparation for that